### PR TITLE
Move EVM config ocr and ocr2

### PR DESCRIPTION
**Pull request by george-dorin**
- Move `OCR` and `OCR2` EVM chain scoped config

### CODE REVIEW

**Review by cedric-cordenier — DISMISSED** [review]

File: core/services/job/runner_integration_test.go [hunk 1]
```diff
@@ -1124,3 +1110,5 @@ func TestRunner_Error_Callback_AsyncJob(t *testing.T) {
 		require.Eventually(t, func() bool { return eiNotifiedOfDelete }, 5*time.Second, 10*time.Millisecond, "expected external initiator to be notified of deleted job")
 	}
 }
+
+func ptr[T any](t T) *T { return &t }
```
Comment (jmank88) [inline core/services/job/runner_integration_test.go:1114]: Hoping we can drop these in two release :crossed_fingers: 
https://github.com/golang/go/issues/45624

File: core/chains/evm/config/v2/config_ocr2_test.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/v2/core/internal/cltest"
+)
+
+func TestOcr2Config(t *testing.T) {
+	evmOcrCfg := cltest.NewTestChainScopedConfig(t) //fallback.toml values
+	require.Equal(t, uint32(5300000), evmOcrCfg.EVM().OCR2().Automation().GasLimit())
```
Comment (jmank88) [inline core/chains/evm/config/v2/config_ocr2_test.go:13]: This is testing `ocr2Config` right? In that case, I think the idiomatic Go naming would be:
```suggestion
func Test_ocr2Config(t *testing.T) {
	evmOCRCfg := cltest.NewTestChainScopedConfig(t) //fallback.toml values
	require.Equal(t, uint32(5300000), evmOCRCfg.EVM().OCR2().Automation().GasLimit())
```

File: core/chains/evm/config/v2/config_ocr2.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	c OCR2
+}
+
+type ocr2Automation struct {
+	c Automation
+}
+
+func (o ocr2Automation) GasLimit() uint32 {
+	return *o.c.GasLimit
+}
```
Comment (jmank88) [inline core/chains/evm/config/v2/config_ocr2.go:15]: Order and pointer receiver?
```suggestion
type ocr2Automation struct {
	c Automation
}

func (o *ocr2Automation) GasLimit() uint32 {
	return *o.c.GasLimit
}

type ocr2Config struct {
	c OCR2
}
```

Comment (george-dorin) [reply core/chains/evm/config/v2/config_ocr2.go:15]: Fixed in [`84f922f` (#9606)](https://github.com/smartcontractkit/chainlink/pull/9606/commits/84f922fd3885d4a8ff29121bba8cb0e5597a68ca)

File: core/chains/evm/config/v2/config_ocr2_test.go (hunk 2 above)
Comment (george-dorin) [reply core/chains/evm/config/v2/config_ocr2_test.go:13]: Fixed in [`84f922f` (#9606)](https://github.com/smartcontractkit/chainlink/pull/9606/commits/84f922fd3885d4a8ff29121bba8cb0e5597a68ca)

**Pull request merged**